RHIDP-9010: AEM-related changes (#1426)
redhat-developer/red-hat-developers-documentation-rhdh: modules/customizing-the-appearance/proc-loading-custom-theme-using-dynamic-plugin.adoc

  frontend:
    example.my-custom-theme-plugin:
      themes:
        - id: light
          title: Light
          variant: light
          icon: someIconReference
          importName: lightThemeProvider
----

> `id`:: Set your theme ID by specifying the desired value. Optionally, override the default {product-short} themes by using the following ID values: `light` to replace the default light theme, or `dark` to replace the default dark theme.

Reviewer: ```suggestion
`id`:: Enter your theme ID, such as `my_theme`. 
Enter `dark` to override the default {product-short} dark theme.
Enter `light` to override the default {product-short} light theme.
```
